src/main/resources/static/openapi/EducationAndWorkPlanAPI.yml

```diff
@@ -1599,6 +1599,18 @@ components:
           type: string
           description: The identifier of the prison that the prisoner was resident at when this resource was updated.
           example: 'BXI'
+      required:
+        - reference
+        - prisonNumber
+        - note
+        - createdBy
+        - createdByDisplayName
+        - createdAt
+        - createdAtPrison
+        - updatedBy
+        - updatedByDisplayName
+        - updatedAt
+        - updatedAtPrison
```

Comment:
Not 100% sure why, but this fixed those mapstruct write errors I was encountering. I think maybe when the fields are nullable mapstruct will try to call the constructor with null values then attempt to update the individual fields which fails since they are immutable.